scoutingmanagementsystem/overseer/src/renderer/src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { Routes, Route } from 'react-router-dom';
 import { Capturer } from './components/Capturer';
 import { Distributor } from './components/Distributor';
 
-export const App: React.FC = (): ReactElement => {
+export default function Dashboard() {
```

Comment:
I don't like default exports. It allows for the engineer to choose whichever name they wish. This is bad. The same thing should not have multiple names.